pkg/antctl/transform/networkpolicy/transform.go

```diff
@@ -20,34 +20,72 @@ import (
 	"sort"
 	"strconv"
 
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubectl/pkg/cmd/get"
+	"k8s.io/kubectl/pkg/scheme"
+	"k8s.io/utils/strings/slices"
+
 	"antrea.io/antrea/pkg/antctl/transform"
 	"antrea.io/antrea/pkg/antctl/transform/common"
 	cpv1beta "antrea.io/antrea/pkg/apis/controlplane/v1beta2"
-	"antrea.io/antrea/pkg/controller/networkpolicy"
 )
 
+const sortByEffectivePriority = "effectivePriority"
+
 type Response struct {
 	*cpv1beta.NetworkPolicy
 }
 
+type NPSorter struct {
+	networkPolicies []cpv1beta.NetworkPolicy
+	sortBy          string
+}
+
 func objectTransform(o interface{}, _ map[string]string) (interface{}, error) {
 	return Response{o.(*cpv1beta.NetworkPolicy)}, nil
 }
 
 func listTransform(l interface{}, opts map[string]string) (interface{}, error) {
+	specialCases := []string{
```

Comment:
`specialCases` sound informal and is not very readable. Perhaps name it `customSorters`.

Reply:
OK

Reply:
This was not addressed?
I'm making this "unresolved".